gcp/modules/gpii-flowmanager/main.tf

```diff
@@ -23,21 +24,28 @@ variable "secret_couchdb_admin_username" {}
 
 variable "secret_couchdb_admin_password" {}
 
+locals {
+  user_email = "${var.auth_user_email != "" ? var.auth_user_email : "dev-null@raisingthefloor.org"}"
+  acme_email = "${var.env == "prd" || var.env == "stg" ? "ops@raisingthefloor.org" : local.user_email}"
+}
+
 data "template_file" "flowmanager_values" {
-  template = "${file("values.yaml")}"
+  template = "${file("${path.module}/templates/values.yaml.tpl")}"
 
   vars {
     domain_name            = "${var.domain_name}"
     flowmanager_repository = "${var.flowmanager_repository}"
     flowmanager_checksum   = "${var.flowmanager_checksum}"
     couchdb_admin_username = "${var.secret_couchdb_admin_username}"
     couchdb_admin_password = "${var.secret_couchdb_admin_password}"
-    cert_issuer_name       = "${var.cert_issuer_name}"
     replica_count          = "${var.replica_count}"
     requests_cpu           = "${var.requests_cpu}"
     requests_memory        = "${var.requests_memory}"
     limits_cpu             = "${var.limits_cpu}"
     limits_memory          = "${var.limits_memory}"
+    project_id             = "${var.project_id}"
+    acme_server            = "${var.env == "prd" || var.env == "stg" ? "https://acme-v02.api.letsencrypt.org/directory" : "https://acme-staging-v02.api.letsencrypt.org/directory"}"
```

Comment:
Should this move to Terragrunt level?

Reply:
I've considered this, actually 3 options:
1. move this to Terragrunt, requiring full url -> I didn't like having to ask for full url, easy to make mistakes
2. move this to Terragrunt, but put in some logic so the config is `staging`/`production`, or `use_production_le` with `true/false` -> this would involve more logic for no obvious benefit
3. as it is 

In the end I've decided for 3) - since it's only a binary choice and we can cover the full logic here, I prefer to have self-contained, no config module. Also it's consistent with how the email is handled (and that would not be possible to move to Terragrunt due to the lack of variable interpolation)